src/routes/utils/helpers.js

```diff
@@ -43,3 +46,23 @@ export const verifyToken = async (req, res, next) => {
     return errorResponse(res, 400, error.message);
   }
 };
+
+export const checkUser = async (req, res, email) => {
```

Comment:
lets makes this a normal function that takes email as an argument and returns user if found and/or throws an error if user is not found or return null.

Reply:
this has been done

Reply:
this has been fixed